hummingbird/ml/operator_converters/_base_operator.py

```diff
@@ -6,7 +6,7 @@ class BaseOperator(ABC):
     Abstract class defining the basic structure for operator implementations in Hummingbird.
     """
 
-    def __init__(self, regression=False, classification=False, transformer=False, anomaly_detection=False, **kwargs):
+    def __init__(self, regression=False, classification=False, transformer=False, anomaly_detection=False, input_indices=None, append_output=False, **kwargs):
```

Comment:
Sorry but can you add some documentation for the class? (the signature is a bit bloated and I think it makes sense to have documentation to take track of the different options)